app/presenters/piv_cac_recommended_presenter.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class PivCacRecommendedPresenter
+  attr_accessor :user
```

Comment:
Minor: If we're not expecting to reassign, `attr_reader` could suffice

```suggestion
  attr_reader :user
```